libraries/AP_HAL_PX4/Util.cpp

```diff
@@ -27,6 +27,7 @@ extern bool _px4_thread_should_exit;
  */
 PX4Util::PX4Util(void) : Util()
 {
+    _system_time_was_set = false;
```

Comment:
Will this object ever exist on the stack or in some other way not be initialised to all-zeroes?

If not, this line should be removed.